src/mod_pvp_titles.cpp

```diff
@@ -55,7 +55,7 @@ class PVPTitles : public PlayerScript
                         Killer->SetTitle(sCharTitlesStore.LookupEntry(15));
                     }
                 }
-                else
+                if (team == TEAM_ALLIANCE)
```

Comment:
if you put `else if` instead of just `if` you save the double check on the variable `team` in case of horde